terracumber/tf_module_cleaner.py

```diff
@@ -21,9 +21,9 @@ def get_default_modules(maintf_content, tf_resources_to_delete):
 
     if tf_resources_to_delete:
         if 'retail' in tf_resources_to_delete:
-            exclusions.extend(['terminal', 'buildhost'])
+            exclusions.extend(['terminal', 'buildhost','proxy','dhcp_dns'])
         if 'proxy' in tf_resources_to_delete:
-            exclusions.append('proxy')
+            exclusions.extend(['proxy','dhcp_dns'])
```

Comment:
just a nitpick: to be consistent, put a space after the commas

Reply:
I don't see the reason to include dhcp_dns as a module to don't be deleted in the 'proxy' statement. It make sense in 'retail' statement only.